posthog/api/dashboards/dashboard_templates.py

```diff
@@ -84,7 +84,16 @@ def repository(self, request: request.Request, **kwargs) -> response.Response:
     def _load_repository_listing() -> List[Dict]:
         url = "https://raw.githubusercontent.com/PostHog/templates-repository/main/dashboards/dashboards.json"
         loaded_templates: List[Dict] = json.loads(requests.get(url).text)
-        return loaded_templates
+        return [
+            # The OG template is hard-coded, not served by the repository, so added in to the template listing here
+            {
+                "name": "Product analytics",
+                "url": "some url",
```

Comment:
Feels wrong to have "some url" as a string in production code…

Reply:
🙈